### PR TITLE
Add quay.io to scraped statuses

### LINKED ISSUE #103: add quay.io to the status page
Then we won't need to open issues here.

### PR DESCRIPTION
**Pull request by chrisburr**
No idea if this is correct but if it is, closes #103

### CODE REVIEW

**Comment by beckermr** [comment]
the div needs to be added to the template - i'll push the correct thing

**Comment by chrisburr** [comment]
I think I did it as you were writing the comment 😄

File: site/cistatus.js [hunk 1]
```diff
@@ -47,3 +47,4 @@ loadCIHTML('https://www.githubstatus.com/', 'GitHub', 'github', displayStatus)
 loadCIHTML('https://www.traviscistatus.com', 'Travis CI', 'travis', displayStatus)
 loadCIHTML('https://status.circleci.com', 'CircleCI', 'circle', displayStatus)
 loadCIHTML('https://status.appveyor.com', 'AppVeyor', 'appveyor', displayStatus)
+loadCIHTML('https://status.quay.io/', 'quay.io', 'quay', displayStatus)
```
Comment (beckermr) [inline site/cistatus.js:50]:
```suggestion
loadCIHTML('https://status.quay.io/', 'Quay.io', 'quay', displayStatus)
```

File: site/template.html [hunk 2]
```diff
@@ -184,6 +184,7 @@ <h4 id="cloud_services"><a href="#cloud_services" style="color: inherit;">Cloud
         <div id="ciStatusDiv-travis"><h5><a href="https://www.traviscistatus.com">Travis CI status</a>: No Status Available</h5></div>
         <div id="ciStatusDiv-circle"><h5><a href="https://status.circleci.com">CircleCI status</a>: No Status Available</h5></div>
         <div id="ciStatusDiv-appveyor"><h5><a href="https://status.appveyor.com">AppVeyor status</a>: No Status Available</h5></div>
+        <div id="ciStatusDiv-quay"><h5><a href="https://status.quay.io/">quay.io status</a>: No Status Available</h5></div>
```
Comment (beckermr) [inline site/template.html:187]:
```suggestion
        <div id="ciStatusDiv-quay"><h5><a href="https://status.quay.io/">Quay.io status</a>: No Status Available</h5></div>
```

**Comment by beckermr** [comment]
Thanks @chrisburr !

**Pull request merged**